commands/stop.go

```diff
@@ -1,13 +1,23 @@
 package commands
 
 import (
+	"fmt"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lotus/api"
 	lotuscli "github.com/filecoin-project/lotus/cli"
 	"github.com/urfave/cli/v2"
 )
 
 var StopCmd = &cli.Command{
 	Name:  "stop",
 	Usage: "Stop a running lily daemon",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:   "height",
+			Usage:  "Get the last chain height during the given sync stage. One of [header, message]",
+			Hidden: false,
```

Comment:
Adding this explicitly because I want to put this up for discussion. We could potentially keep it hidden so that Lily users are not necessarily encouraged to use this.